changelog.d/38.misc

```diff
@@ -0,0 +1 @@
+Fix email support.
```

Comment:
please could you expand on this a bit? this implies it was completely broken before, which I don't think is the case?

(also, this should be a `.bugfix` file)

Reply:
will rename the file and try to expand the description. Although mailing was not working for me at all. But it may depend on the setup